<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateObjectBusDialog</class>
 <widget class="QDialog" name="CreateObjectBusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>model:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>year:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>price:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>registration number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>seats count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>has disabled seats:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="idLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="modelLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="yearLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="priceLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="regNumLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="seatsCountLineEdit"/>
         </item>
         <item>
          <widget class="QCheckBox" name="hasDisabledSeatsCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="createObjectBusButton">
       <property name="text">
        <string>Create Object</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
